resdk/analysis/prepare_geo.py

```diff
@@ -43,8 +43,7 @@ def prepare_geo_chipseq(resource, name=None):
 
     """
     reads = []
-    macs14 = []
-    relations = []
+    macs = []
```

Comment:
Don't forget to add a change log.

Reply:
This was just a "technical fix". The sinature of the function `prepare_geo_chipseq` did not change. So for the end user, there is no visible difference. AFAIK such changes don't belong to changelog...

Reply:
Ok, it makes sense.